ui/app/AppLayouts/Chat/controls/community/HoldingsDropdown.qml

```diff
@@ -0,0 +1,277 @@
+import QtQuick 2.13
+import QtQuick.Layouts 1.14
+import QtQuick.Controls 2.13
+import QtGraphicalEffects 1.13
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Components 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Controls.Validators 0.1
+import StatusQ.Core.Utils 0.1 as SQ
+
+import utils 1.0
+
+StatusDropdown {
+    id: root
+
+    property real tokenAmountValue: 0
+    property string tokenName: d.defaultTokenNameText
+    property url tokenImage: ""
+    property int operator: SQ.Utils.Operators.None
+    property bool withOperatorSelector: true
+
+    signal addToken(string tokenText, url tokenImage, int operator)
+
+    function reset() {
+        root.tokenAmountValue = 0
+        root.tokenName = d.defaultTokenNameText
+        root.tokenImage = ""
+        root.operator = SQ.Utils.Operators.None
```

Comment:
Not need to reset withOperatorSelector ?

Reply:
> At minimum height the create button is clipped

Here the task that will include the issue 1: https://github.com/status-im/status-desktop/issues/6042

Reply:
Thanks for pointing it out. No, `withOperatorSelector` is a configuration of the component and wants to be kept although reset is done.